cookbooks/aws-parallelcluster-config/resources/manage_fsx.rb

```diff
@@ -133,6 +133,7 @@
       command "umount #{fsx_shared_dir}"
       retries 10
       retry_delay 6
+      only_if "mountpoint -q #{fsx_shared_dir}"
```

Comment:
The role of this `only_if` is to make the resource idempotent.
Afaik we do not have this in `manage_efs` resource.
Shouldn't we put that there, too?

Reply:
Added also for other resources